components/Shipping/exceptions.rst

```diff
@@ -0,0 +1,38 @@
+Exceptions
+==========
+
+Calculator Registry Exceptions
+------------------------------
+
+ExistingCalculatorException
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This exception should be thrown by calculator registry when calculator with given name already exists.
```

Comment:
Since it's already under ``Calculator Registry Exceptions`` title, think you could drop the `by` statement and simply go with `This exception is thrown when ...`